src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,31 @@
-export const TodoInfo = () => {};
+import cn from 'classnames';
+
+import { Todo } from '../../types';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo,
+};
+
+export const TodoInfo: React.FC<Props> = ({
+  todo: {
+    id,
+    title,
+    completed = false,
+    user,
+  },
```

Comment:
As mentioned [mykhailoyurkovskyi](https://github.com/mykhailoyurkovskyi)
It's better to do destructuring in the body of the component.

`const { id, title, completed, user } = todo;`

Also, completed is a required field in the `Todo` interface so you don't need to set default value